FlashCards.AhmadJer99/FlashCards/Managers/CardsManager.cs

```diff
@@ -0,0 +1,160 @@
+﻿using FlashCards.Models;
+using FlashCards.Dtos;
+using FlashCards.Data;
+using FlashCards.Mappers;
+using Spectre.Console;
+
+namespace FlashCards.Managers;
+
+internal class CardsManager : ModelManager<Card>
+{
+    public enum CardOperation
+    {
+        ShowCardsInStack,
+        AddCardToStack,
+        SeeAmountOfCardsInStack,
+        DeleteCard,
+    }
+    private int currentStackID;
```

Comment:
🟠 **Naming Conventions**

💡 Like database, identifier is one word, and commonly abbreviated to ID. But if you are heart set on `ID`, make it consistent. You have a method called `LoadCardsWithStackId`.

🔧 Rename to `currentStackId` or rename all other instances of `*Id` to `*ID`.

Reply:
Done fixed all id instances and changed it to Id